.github/workflows/image.yaml

```diff
@@ -18,6 +18,40 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
+  build-and-push-devbox-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    name: Build and Push app-autoscaler-release-devbox
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-devbox
```

Comment:
Why not adding an additional env var vor the devbox image name

```yaml
env:
  IMAGE_NAME_DEVBOX: ${{ github.repository }}-devbox
```

And then referring to it 
```yaml
      - name: Extract metadata (tags, labels) for Docker
        id: meta
        uses: docker/metadata-action@v5
        with:
          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME_DEVBOX }}
```

Reply:
fixed in 6d493161a94540804b870b403c7dd3e7d49f61d2